packages/shared/src/styles/media.ts

```diff
@@ -7,4 +7,3 @@ export const laptopL = `@media (min-width: 1360px)`;
 export const laptopXL = `@media (min-width: 1668px)`;
 export const desktop = `@media (min-width: 1976px)`;
 export const desktopL = `@media (min-width: 2156px)`;
-export const userPrefersReducedMotions = `@media (prefers-reduced-motion)`;
```

Comment:
@omBratteng are we not using this in any animation we have at the moment? Or we are using something else now?